packages/coil-extension/src/background/services/utils.ts

```diff
@@ -61,3 +61,19 @@ export function createLogger(context: Context) {
     `color: white;`
   )
 }
+
+export function logLastError(from: string): () => void {
```

Comment:
This seems to log an error if the other side closes the channel immediately, which is quite often the case in unidirectional messaging.